src/cache/inmemory/reactiveVars.ts

```diff
@@ -64,7 +81,22 @@ export function makeVar<T>(value: T): ReactiveVar<T> {
     };
   };
 
-  return rv;
+  if (!config) return rv;
+
+  const restore = async () => {
+    // Set reactiveVar to previous value from storage,
+    // if there is no previous value, do nothing.
+    try {
+      const previousValue = await config.storage.getItem(config.storageKey)
+      if (previousValue) {
+        rv(isString(value) ? previousValue : JSON.parse(previousValue))
```

Comment:
Do you mean to be checking `previousValue` here instead?


```suggestion
        rv(isString(previousValue) ? previousValue : JSON.parse(previousValue))
```

Reply:
Thanks @rathpc, but here I really meant the `value`. We need to know if the original value is string, because the `previousValue` always is string, since it came from the local storage (as string).

Reply:
The problem here would be if the type of the value is a union of `string` and something not a string.

Reply:
I see, ok makes sense!